### PR TITLE
Removed classic patch map option, and deleted relevant code.

### LINKED ISSUE #4200: Remove the classic map option as it has extra bugs (and no one really seems to want to work on it)
just so that there's no trap in the new game settings for getting a worse experience.

This one bug was open for a really long time without anyone giving it a thought really:
- https://github.com/Revolutionary-Games/Thrive/issues/3527

No one (at least on the dev team) seems to play the classic map that often so I think it'll be overall better to just remove the option instead of hoping that we don't break it anymore with further game changes.

### PR DESCRIPTION
**Pull request by Creidhne86**
I removed the classic patch map from the game. Translations were also updated based on the removal.

closes #4200 

Other issues: #4195, #3527 

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Initial code review passed (this and further items should not be checked by the PR author)
- [x] Functionality is confirmed working by another person (see above checklist link)
- [x] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by hhyyrylainen** [comment]
This needs to have the CI check automatically found problems fixed. Do you need help with those?
Jetbrains is complaining about an extra blank line between an XML comment and the code element it describes. Format script fails because the localization script has not been ran to update things.

File: src/microbe_stage/BiomeType.cs [hunk 1]
```diff
@@ -1,12 +1,7 @@
 ﻿/// <summary>
 ///   This contains predefined biome types
 /// </summary>
-/// <remarks>
-///   <para>
-///     The order of this enum CANNOT be changed. They are used as patch IDs in classic map,
```
Comment (hhyyrylainen) [inline src/microbe_stage/BiomeType.cs:6]: This comment actually needs to be kept, but reworded to say that the order of the enum values may not be changed due to saves saving the values. So reordering the values will break all existing saves, which is very bad.

**Comment by Creidhne86** [comment]
> 

I wasn't aware of the translate update.. I found this documentation and I'll ask questions if I'm not able to figure it out.

https://github.com/Revolutionary-Games/Thrive/blob/master/doc/working_with_translations.md

**Comment by Creidhne86** [comment]
@hhyyrylainen seems like everything is passing now, thanks for the help!

**Review by hhyyrylainen — APPROVED** [review]
I tested this and didn't see any Godot errors or leftover GUI for the feature to be removed. I noticed one typo reported by my IDE which I fixed, so this should be all good to go now.

**Pull request merged**